dragonchain/broadcast_processor/broadcast_processor.py

```diff
@@ -110,6 +120,74 @@ def make_broadcast_futures(session: aiohttp.ClientSession, block_id: str, level:
     return broadcasts
 
 
+def get_level_from_storage_location(storage_location: str):
```

Comment:
should annotate return type here for completeness. Would be `Optional[str]`

Reply:
good idea. changing